src/components/Reports/Subscribers/Subscribers.js

```diff
@@ -98,8 +99,7 @@ const Subscribers = ({ dependencies: { dopplerApiClient } }) => {
           <p>{currentSection.description}</p>
         </div>
       </HeaderSection>
-
-      <section className="dp-container">
+      <ContentSection cssClass="dp-container">
```

Comment:
Why not include this class 'dp-container' inside `ContentSection component`?

Reply:
i talked with @GusBaamonde, because don't need always, there are cases we dont need in the new pages

Reply:
We have three cases of backgrounds, white page, gray page and as the Doppler plus case they use both at the same time. The class "dp-container" is to use the grid, we will not always want or need to use the grid at that point.

Reply:
Pero que ganamos con este `ContentSection`? si al fin y al cabo es lo mismo que un `section`?

Reply:
Quizás lo estamos pensando de la forma incorrecta. Lo que estamos queriendo hacer es armar componente para poder hacer un armado rápido de layouts, sean grises, blancos etc con una estructura mínima. Como el que se usa en reportes, un header gris y la parte de abajo blanca. Quizás el problema es como lo estamos haciendo. No me parece mal que tengamos la opción de armar layout rápidamente si tener que hacerlo a manopla todas las veces.

Reply:
En el objetivo estoy de acuerdo. Pero me parece que el componente `ContentSection` (al menos hasta ahora) es lo mismo que un `section` y en ese caso, prefiero el `section` original.

Tal vez ustedes estan viendo un poco más allá como va a evolucionar esto, si es así, estaría bueno incluir eso en este mismo PR porque deberían ser evidentes las ventajas de hacer este cambio antes de hacerlo.

Reply:
Ahi nos juntamos con los chicos y definimos que por ahora no tiene sentido este componente si va ser solo un section, luego en el futuro si vale la pena se vera si se hara.